zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1991,25 +2041,58 @@ components:
       properties:
         flowNodeInstanceKey:
           type: integer
-          description: The key of this flow node instance.
+          description: The assigned key, which acts as a unique identifier for this flow node instance.
           format: int64
         processInstanceKey:
           type: integer
-          description: The process instance key.
+          description: The process instance key associated to this flow node instance.
           format: int64
         processDefinitionKey:
           type: integer
-          description: The process definition key.
+          description: The process definition key associated to this flow node instance.
           format: int64
+        bpmnProcessId:
+          description: The bpmn process id associated to this flow node instance.
+          type: string
         state:
+          description: State of flow node instance as defined set of values.
           type: string
-          description: The state, one of ACTIVE, COMPLETED, TERMINATED.
+          enum:
+            - ACTIVE
+            - COMPLETED
+            - TERMINATED
         type:
+          description: Type of flow node as defined set of values
```

```suggestion
          description: Type of flow node as defined set of values.
```

:x: Sentence-case, for consistency.

🔧 I also notice that there are other properties in this schema that are lacking a period, though they are not currently modified in this PR. What do you think about adjusting those also? (`flowNodeName`, `treePath`, `incident`, etc.)

---

Thank you @pepopowitz, I'll take a look at those too.